### DISCUSSION TITLE
API usage

### DISCUSSION THREAD

**Discussion by notmeta** [ory/kratos#1364] [Q&A] [answered]
So I was reading through [recent changes](https://github.com/ory/kratos/blob/master/CHANGELOG.md#063-alpha1-2021-05-17) and noticed 'ViaAPIFlow' being renamed to 'ForNativeApps'.
Is Kratos moving away from offering support for APIs, in favour of 'native apps'?

It would be pretty disappointing if that was the case, perhaps I'm just reading too far into it or misunderstanding something.

I am currently looking into implementing user login via the API on-behalf of users. My use case is users (outside of the website, which will use browser flows for login/registration) can login via a game client and I plan on forwarding that request on behalf of the user via API flows. Is this workflow supported, or should I be doing something different?

It would be ideal if this was supported, as re-implementing login elsewhere kind of defeats the purpose of using Kratos!

Thanks! :smile:

**Comment by aeneasr** [comment, marked as answer]
The flow doesn't change, it's just a renaming. Could probably use some clarification. Native Apps are:

- CLIs
- Games
- Services
- Mobile Apps
- Desktop Apps
- ...

**Reply by notmeta** [reply]
Great! Thanks for the speedy response :slightly_smiling_face:
